osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -154,12 +152,18 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
             CurrentNumber.BindValueChanged(current => TooltipText = getTooltipText(current.NewValue), true);
-            Current.DisabledChanged += disabled => hoverClickSounds.Enabled.Value = !disabled;
+
+            Current.BindDisabledChanged(disabled =>
+            {
+                Alpha = disabled ? 0.3f : 1;
+                hoverClickSounds.Enabled.Value = !disabled;
+            }, true);
         }
 
         protected override bool OnHover(HoverEvent e)
         {
-            updateGlow();
+            if (!Current.Disabled)
+                updateGlow();
```

Comment:
This is also an issue in `OnHoverLost()` and `OnDragEnd()`. May be a better idea to move the `Current.Disabled` guard to inside `updateGlow()`.